aten/src/ATen/Context.cpp

```diff
@@ -290,7 +290,11 @@ at::BlasBackend Context::blasPreferredBackend() {
 #ifdef USE_ROCM
   if (blas_preferred_backend == at::BlasBackend::Cublaslt) {
     static const bool hipblaslt_unsupported = []() {
+#if ROCM_VERSION >= 60200
+      static const std::vector<std::string> archs = {"gfx90a", "gfx940", "gfx941", "gfx942", "gfx1100", "gfx1101"};
+#else
       static const std::vector<std::string> archs = {"gfx90a", "gfx940", "gfx941", "gfx942"};
+#endif
```

Comment:
Could remove some repetition?

```suggestion
    static const std::vector<std::string> archs = {"gfx90a", "gfx940", "gfx941", "gfx942", 
#if ROCM_VERSION >= 60200
    "gfx1100", "gfx1101",
#endif
    };
```

Reply:
Great suggestion.  Made the same change in both places.